modules/ROOT/pages/dw-arrays-functions-join.adoc

```diff
@@ -0,0 +1,83 @@
+= join
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+
+[[join1]]
+== join&#40;Array<L&#62;, Array<R&#62;, &#40;leftValue: L&#41; &#45;&#62; String, &#40;rightValue: R&#41; &#45;&#62; String&#41;: Array<Pair<L, R&#62;&#62;
+
+Joins two array of objects by a given `ID` criteria.
+It will return an Array of object containing only the ones that the `ID` of the `left` can be matched with an `ID` of the `right`.
```

Comment:
Should be present tense unless that would make it inconsistent with existing documentation.